packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -923,14 +923,19 @@ export class PHP implements Disposable {
 		return FSHelpers.isDir(this[__private__dont__use].FS, path);
 	}
 
-	/**
-	 * Checks if a file (or a directory) exists in the PHP filesystem.
-	 *
-	 * @param  path - The file path to check.
-	 * @returns True if the file exists, false otherwise.
-	 */
-	fileExists(path: string) {
-		return FSHelpers.fileExists(this[__private__dont__use].FS, path);
+	/** @inheritDoc */
+	isFile(path: string): boolean {
+		return FSHelpers.isFile(this[__private__dont__use].FS, path);
+	}
+
+	/** @inheritDoc */
+	fileExists(path: string): boolean {
+		try {
```

Comment:
Why change this method?

Reply:
> Why change this method?

This was unintentional and may have been an artifact of cherry-picking an older changeset to kick of this PR.

Sorry about that. I'll fix it.